test/Feature/QuerySpec.hs

```diff
@@ -85,6 +85,19 @@ spec actualPgVersion = do
         [json| [{"id": 3, "name": "wash the dishes", "done": null }] |]
         { matchHeaders = [matchContentTypeJson] }
 
+    it "matches with trilean values in upper or mixed case" $ do
```

Comment:
Maybe it needs a test for `NULL` (not here, but in the "matches nulls..." section) for completion.